quiche/src/recovery/mod.rs

```diff
@@ -321,37 +322,33 @@ impl Recovery {
 
     pub fn set_pacing_rate(&mut self, rate: u64) {
         if rate != 0 {
+            self.pacer.update(self.send_quantum, rate);
+
             self.pacing_rate = rate;
```

Comment:
Do we still need this field? Looks like it is only used in `schedule_next_packet()` to check `pacing_rate == 0`, and we could just use `pacer.pace` instead (e.g. adding a `rate()` method to `Pacer`).

Reply:
`Recovery.pacing_rate` is removed and replaced by `pacer.rate()`.